src/Sylius/Bundle/ShopBundle/templates/product/show/page/product_listing/associations.html.twig

```diff
@@ -1,8 +1,7 @@
 {% set product = hookable_metadata.context.product %}
 
 {% for association in product.associations %}
-    {{ render(url('sylius_shop_partial_product_association_show', {'productId': product.id, 'id': association.id, 'template': '@SyliusShop/product/show/page/product_listing/association.html.twig'})) }}
+    {{ component('sylius_shop:product:association', { productAssociation: association }) }}
 {% else %}
-    <div>{{ 'sylius.ui.latest_products'|trans }}</div>
-    {{ render(url('sylius_shop_partial_product_index_latest', {'count': 4, 'template': '@SyliusShop/shared/product/horizontal_list.html.twig'})) }}
+    {{ component('sylius_shop:homepage:latest_products', { limit: 4 }) }}
```

Comment:
It should definitely change name and be moved to common directory

Reply:
It will be done in other PR